src/components/ebay-combobox/component.js

```diff
@@ -84,8 +90,14 @@ module.exports = {
     onInput(input) {
         input.autocomplete = input.autocomplete === 'list' ? 'list' : 'none';
         input.options = input.options || [];
-        this.lastValue = input.value;
-        this.state = { currentValue: input.value };
+        let inputId;
+        if (input['*']) {
+            this.lastValue = input['*'].value;
+            inputId = input['*'].id;
+        } else {
+            this.lastValue = input.value;
+        }
+        this.state = { currentValue: this.lastValue, id: inputId };
```

Comment:
Where is `currentValue` used?

Reply:
In `_setSelectedText()`, `_emitComboboxEvent()`, and a couple other places.
```
❯ rg currentValue[^a-zA-Z] src/components/ebay-combobox/component.js
55:            this.state.currentValue = this.getEl('combobox').value;
80:        if (this.lastValue !== this.state.currentValue) {
81:            this.lastValue = this.state.currentValue;
100:        this.state = { currentValue: this.lastValue, id: inputId };
161:        if (this.state.currentValue !== text) {
163:            this.state.currentValue = input.value = text;
183:        const currentValueReg = safeRegex(this.state.currentValue);
193:            currentInputValue: this.state.currentValue,
```

Reply:
Got it! Looks good

I would remove the id from state and just use the ID directly from input in the template.